bogoworld/main.py

```diff
@@ -0,0 +1,58 @@
+from random import randint
+import sys, time
+
+#prints a string by:
+# printing a random character
+# checking against the current desired character
+# backspacing if the result was incorrect
+#
+#Arguments:
+# s - the string to be printed (ASCII-only)
+# d - the average delay between correctly printed characters (in seconds)
+def bogoprint(s, delay=0.2):
+
```

Comment:
Consider using a docstring and more descriptive variables:

```suggestion
def bogoprint(print_str, delay=0.2):
    '''
    prints a string by printing and backspacing a random character until
    the correct character is output

    print_str (str) : ASCII-only print string
    delay (float)   : average seconds between correctly printed characters
    '''
```